index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Eternalgratis</title>
+</head>
+<body>
+    <header>
+       <div class="header">
+           <a href="index.html">Welcome</a>
+           <img src="images/menu.png" alt="">
+       </div> 
+    </header>
+    <section class="landing-page">
+        <div class="lp-container">
+            <h1>Hey there. I'm Titilope. <br>
+              <span class="tint">I'm a software developer.</span>
+            </h1>  
+            <p class="lp-text text"> I can help you build a product, feature or website Look through 
+                some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.
+            </p>
+            <div class="media-links">
+                <ul class="social">
+                    <li><a href=""><img src="images/twitter.svg" alt="my twitter profil"></a></li>
+                    <li><a href=""><img src="images/linked.svg" alt="connect on linkedin"></a></li>
+                    <li><a href=""><img src="images/medium.svg" alt="check medium"></a></li>
+                    <li><a href=""><img src="images/github.svg" alt="github profile"></a></li>
+                    <li><a href=""><img src="images/angel.svg" alt="angel list"></a></li>
+                </ul>
```

Comment:
You did great for grouping the icons with the `ul` tag and then using `li` for each icon and also including the `a` tag.
I like that, kudos 💯